mineinabyss-features/src/main/kotlin/com/mineinabyss/features/gondolas/GondolasHelpers.kt

```diff
@@ -0,0 +1,47 @@
+package com.mineinabyss.features.gondolas
+
+import com.mineinabyss.components.gondolas.Gondola
+import org.bukkit.entity.Player
+import org.bukkit.Location
+import kotlin.math.abs
+
+enum class GondolaType() {
+    UPPER,
+    LOWER,
+    NONE;
+}
+
+fun gondolaWarp(gondola: Gondola, player: Player, gondolaType: GondolaType) {
+    val loc =
+        if (gondolaType == GondolaType.LOWER) gondola.upperLoc else gondola.lowerLoc
+    player.teleport(loc)
+    player.sendMessage("You have been warped to ${gondola.name} at ${loc.x}, ${loc.y}, ${loc.z}")
+}
+
+// AABB detection
+// returns if locations contains point within radius
+fun locContains(loc: Location, point: Location, radius: Double): Boolean {
+    return abs(loc.x - point.x) <= radius &&
```

Comment:
there is definetly some method for this, though maybe not with radius in mind, check out AABB or somerthing iirc, Paper has some api, can check their javadocs too